h/templates/app.html.jinja2

```diff
@@ -52,6 +52,25 @@
       <main ng-view=""></main>
     </div>
 
+    <div class="loggedout-message" ng-if="isSidebar && auth.status === 'signed-out' && settings.annotations" ng-cloak>
+      This is a public annotation created with Hypothes.is
```

Comment:
Re-reading the description on the card, this message should only be shown if:

1. There is a selection
2. The user is not signed in AND
3. The annotation is available to the user

Together these imply that the annotation must exist and it must be public.
This can be implemented in a very similar way to the implementation of https://github.com/hypothesis/h/pull/3093 .

I suggest rebasing on master, then moving this to `viewer.html` and changing the `ng-if` condition to be similar to the way that the permission message is displayed.